lib/open_api_spex/cast/primitive.ex

```diff
@@ -0,0 +1,68 @@
+defmodule OpenApiSpex.Cast.Primitive do
+  @moduledoc false
+  alias OpenApiSpex.Cast
+  alias OpenApiSpex.Cast.String
+
+  def cast(%{schema: %{type: :boolean}} = ctx),
```

Comment:
Is there a benefit to having all these value type functions bundled up into their own module versus having the public casts from this module living in `OpenApiSpex.Cast` and the private casts living in their own modules under the `Cast` namespace? The guard clause at https://github.com/GhostGroup/open_api_spex/pull/1/files#diff-dd68d9c14f40d7c7a8044180a42f535cR56 I believe is compiled to 4 pattern matched heads, all of which would have the same pattern match as represented in these public cast calls.

Reply:
I'm not following completely. Let's talk when you're online.

Reply:
Never mind. I see what you're saying. I just pushed up a commit that should address what you're asking.